src/UI/Component/Alert/Standard/Factory.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/* Copyright (c) 2018 Thomas Famula <famula@leifos.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Alert\Standard;
+
+/**
+ * This is how a factory for Standard Alerts looks like.
+ */
+interface Factory {
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     The system failed to complete some actions and displays information about the failure (brand-danger).
+	 *
+	 * rules:
+	 *   interaction: >
+	 *     Failure Standard Alerts MUST NOT be interactive.
```

Comment:
Why should a failure alert not contain a link to some page where I could fix it/restart it/...?

Reply:
All change to "may" use Buttons now...